src/ADAL.PCL.Desktop/CryptographyHelper.cs

```diff
@@ -145,5 +139,90 @@ private static RSACryptoServiceProvider GetCryptoProviderForSha256(RSACryptoServ
 
             return rsaProvider;
         }
+
+
+        /// <summary>
+        ///     <para>
```

Comment:
I don't think that the first 2 para apply to this method? Is this the standard XML comment syntax?